net.poedit.Poedit.json

```diff
@@ -51,6 +50,49 @@
                     ]
                 }
             ]
+        },
+                {
+            "name": "webkitgtk",
+            "config-opts": [
+                "--with-gtk=2.0",
```

Comment:
also I don't get why you'd build webkit gtk for gtk-2.0 when wxwidget is built for gtk3. Also why do you need to build webkit? It's in the platform.